packages/fxa-content-server/app/scripts/lib/app-start.js

```diff
@@ -428,6 +428,18 @@ Start.prototype = {
   },
 
   createView (Constructor, options = {}) {
+    const {
+      subscriptions: {
+        allowedLanguages: subscriptionsManagementLanguages = ['en-US'],
+        enabled: subscriptionsEnabled = false,
+      } = {},
+      featureFlags: {
```

Comment:
I'd not worry about the featureFlags for this PR, do that in a follow on.

Reply:
I added this because we talked in last retro about starting to use feature flags for unfinished stuff like this.

What would the follow-on PR look like? Wondering if I'm missing things for the feature flag that would complicate it.

Reply:
> I added this because we talked in last retro about starting to use feature flags for unfinished stuff like this.

Yeah, understood. One of the main intentions is so we can modify the config mid-train w/o requiring a re-spin of the stack. Normal config requires a full re-spin w/ the new config, feature flagging no. I'm assuming this won't ship the feature mid-train, at least not in the train this lands on, so I'm suggesting pulling it out for now, because...

> What would the follow-on PR look like?

Because I don't actually know what the feature flagging code should look like and don't want to hold up this PR while trying to figure it out. :/

Reply:
Okie doke! I also just remembered that `SUBSCRIPTIONS_ENABLED` in the overall config defaults to false and also acts to hide this button. So, it's practically a feature flag, too - albeit a deploy-controlled one. But, I think that's fine for this.